tests/conftest.py

```diff
@@ -28,6 +29,13 @@ def client():
     return client
 
 
+@pytest.fixture(scope='module')
+def static_client():
+    client = Client()
+    client._make_request = partial(client._make_request, extra_headers={'Prefer': 'dynamic=false'})
```

Comment:
Will this break if we decide to use `extra_headers` in the Client code in the future? I suspect it will override whatever we could decide to use as `extra_headers`. Would a better implementation be to just append this as an additional header to whatever is sent as `extra_headers` in the function call?

Untested code suggestion:

```suggestion
    client._make_request = partial(client._make_request, extra_headers={'Prefer': 'dynamic=false'})
    client._make_request = lambda *args, **kwargs: client._make_request(*args, **{**kwargs, 'extra_headers': {**kwargs.get('extra_headers', {}), 'Prefer': 'dynamic=false'}})
```